processor/dnslookupprocessor/metadata.yaml

```diff
@@ -0,0 +1,8 @@
+type: dnslookup
+
+status:
+  class: processor
+  stability:
+    development: [traces, metrics, logs]
+  codeowners:
+    active: [andrzej-stencel, edmocosta]
```

Comment:
@kaisecheng as the author of the component, you should also add yourself as one of the codeowners.

Reply:
@andrzej-stencel thanks for being a sponsor of this processor. I removed my name from codeowners because I am not a member, and it was causing the CI check to fail.

Could you support me to be a member? Here is my contribution so far https://github.com/open-telemetry/opentelemetry-collector-contrib/pulls?q=is%3Apr+author%3Akaisecheng+

Reply:
fixed it